lib/web/app.js

```diff
@@ -49,7 +52,10 @@ function buildAuthMiddleware(config, read) {
       dest += '?next=' + req.url;
     }
 
-    if (req.session && req.session.authed === true) {
```

Comment:
This completely overrides our authentication with passport based authentication and will lock us out. Also look at L118-L120 and old L95 vs L143